src/main/java/io/apicurio/datamodels/asyncapi/v2/models/Aai20OperationTrait.java

```diff
@@ -24,41 +16,37 @@ public Aai20OperationTrait(String opType) {
         super(opType);
     }
 
+    /**
+     * Constructor.
+     * @param parent
+     * @param opType
+     */
     public Aai20OperationTrait(Node parent, String opType) {
         super(parent, opType);
     }
 
+    /**
+     * Constructor.
+     * @param parent
+     */
     public Aai20OperationTrait(Node parent) {
         super(parent);
     }
-
-    @Override
-    public Aai20ExternalDocumentation createExternalDocumentation() {
-        return new Aai20ExternalDocumentation(this);
-    }
-
-    @Override
-    public void accept(IVisitor visitor) {
-        IAai20Visitor v = (IAai20Visitor) visitor;
-        v.visitOperationTrait(this);
-    }
-
+    
+    /**
+     * @see io.apicurio.datamodels.asyncapi.models.AaiOperationBase#createTag()
+     */
     @Override
-    public List<AaiProtocolInfo> getProtocolInfoList() {
-        return JsonCompat.mapToList(protocolInfo);
+    public AaiTag createTag() {
```

Comment:
How is this method (and similar) used? Duplicates a method in AaiNodeFactory.

Reply:
I think the node factory is only used by the reader.  We have create methods on the objects themselves for consumption by "users". :)  I believe that is the pattern you have established for asyncapi.